docs/detections/alerts-ui-manage.asciidoc

```diff
@@ -2,29 +2,29 @@
 [role="xpack"]
 == Manage detection alerts
 
-The Detections page displays all <<detection-alert-def, detection alerts>>.
+The Alerts page displays all <<detection-alert-def, detection alerts>>.
 From the Alerts table, you can filter alerts, change an alert's status, and start
 investigating and analyzing alerts in Timeline.
 
-TIP: From Timeline, you can <<cases-ui-open, create cases>> to track issues and
+TIP: From *Timelines*, you can <<cases-ui-open, create cases>> to track issues and
 share information with colleagues.
 
 [float]
 [[detection-view-and-filter-alerts]]
 === View and filter detection alerts
-The Detections page offers various ways for you to organize and triage detection alerts as you investigate suspicious events. You can:
+The Alerts page offers various ways for you to organize and triage detection alerts as you investigate suspicious events. You can:
 
 * Filter for a specific rule in the KQL bar (for example,
 `signal.rule.name :"SSH (Secure Shell) from the Internet"`).
 
 NOTE: KQL autocomplete for `.siem-signals-*` indices is available on the
-*Detections* and *Rule details* pages, and in Timeline when either `All` or
+*Alerts* and *Rule details* pages, and in Timeline when either `All` or
 `Detection alerts` is selected.
 
-* Use the date and time filter to select a time range that you’re interested in exploring. By default, this filter is set to search the last 24 hours.
-* View detection alerts generated by a specific rule. To do this, click *Manage detection rules*, then click on a rule name in the All rules table. The *Rules detail page* displays a comprehensive view of the rule's details, and alert details are displayed in the Alerts table beneath the Trend histogram.
+* Use the date and time filter to define a specific time range. By default, this filter is set to search the last 24 hours.
+* To view detection alerts generated by a specific rule. To do this, go to *Detect* -> *Rules*, and then select a rule name in the All rules table. The *Rules detail page* displays a comprehensive view of the rule's details, and alert details are displayed in the Alerts table beneath the Trend histogram. Alternatively, you can go to *Detect* -> *Alerts* -> *Manage detection rules*, and then click on a rule name in the All rules table to view the alerts associated with the rule.
```

Comment:
I kind of liked the original beginning of this line ("View detection alerts" instead of "To view detection alerts"), because it matches the form of the other bullets, each starting with an active verb and flowing from the "You can..." lead-in to the list (line 15).